frontend/src/components/common/Resource/ViewButton.tsx

```diff
@@ -14,23 +12,13 @@ export interface ViewButtonProps {
 
 function ViewButton({ item, initialToggle = false }: ViewButtonProps) {
   const [toggle, setToggle] = React.useState(initialToggle);
-  const { t } = useTranslation('resource');
 
   function handleButtonClick() {
     setToggle(toggle => !toggle);
   }
   return (
     <>
-      <Tooltip title={t('View YAML') as string}>
```

Comment:
Adding to what I mentioned before, this gets picked up by i18next-parser, but with the new ActionButton it shouldn't. So the translation is lost unless it exists already (meaning it needs to be parsed from elsewhere).

Reply:
Thanks. Fixed.